src/plotman/job.py

```diff
@@ -149,6 +149,8 @@ def get_running_jobs(logroot, cached_jobs=()):
                             parsed_command = parse_chia_plots_create_command_line(
                                 command_line=proc.cmdline(),
                             )
+                            if proc.parent().name() != 'chia':  # grab thread not parent
+                                continue                            
```

Comment:
This will cause other problems, since jobs created by `plotman plot` will return `plotman` as parent's name